README.md

```diff
@@ -76,6 +83,9 @@ key is present on the host with the database dmp file.
 * populate `importDBdmpFile/failover_TEMPLATE.sh` with
 appropriate values and copy it to `/etc/gocdb/failover.sh`
 
+## /root/importMariaDBdmpFile/
```

Comment:
Does the python script assume it is running in the specific directory? Or are file/directory references either:
- relative to whatever directory the script is "installed" into, or
- absolute references

?

Reply:
I don't believe it makes any assumptions about where it is running or installed. Paths (working dir, archive dir, password file) are configurable and assumed to be absolute, but I guess there's no reason that relative paths wouldn't work.